pkg/gadgets/trace/exec/tracer/bpf/execsnoop.bpf.c

```diff
@@ -261,4 +261,30 @@ int ig_execve_x(struct syscall_trace_exit *ctx)
 	return 0;
 }
 
+SEC("kprobe/security_bprm_check")
+int BPF_KPROBE(security_bprm_check, struct linux_binprm *bprm)
+{
+#ifdef WITH_LONG_PATHS
```

Comment:
Does it make sense to wrap the whole function with the WITH_LONG_PATHS ifdef? I think it would be more efficient

Reply:
Though I'm not sure how we can handle the link part in tracer.go if we'll wrap the whole function

Reply:
Unfortunately, tracer.go relies on `execsnoopObjects` and `execsnoopWithLongPathsObjects` having the same bpf programs and maps. It would require more refactoring if we want to do that, so I would prefer to postpone that to another PR.